packages/@ember/-internals/routing/lib/system/route.ts

```diff
@@ -1102,13 +1106,13 @@ class Route extends EmberObject.extend(ActionHandler, Evented) implements IRoute
     // Assign the route's controller so that it can more easily be
     // referenced in action handlers. Side effects. Side effects everywhere.
     if (!this.controller) {
-      let qp = get(this, '_qp');
-      let propNames = qp !== undefined ? get(qp, 'propertyNames') : [];
+      let qp = get(this, '_qp') as any;
```

Comment:
Any chance we can *not* `as any` this one?

Reply:
Yup.